xbmc/messaging/helpers/DialogHelper.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+*      Copyright (C) 2005-2015 Team Kodi
+*      http://kodi.tv
+*
+*  This Program is free software; you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation; either version 2, or (at your option)
+*  any later version.
+*
+*  This Program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with Kodi; see the file COPYING.  If not, see
+*  <http://www.gnu.org/licenses/>.
+*
+*/
+
+#include "DialogHelper.h"
+#include "messaging/ApplicationMessenger.h"
+
+namespace KODI
+{
+namespace MESSAGING
+{
+namespace HELPERS
+{
+int ShowYesNoDialogText(CVariant heading, CVariant text, CVariant noLabel, CVariant yesLabel, uint32_t autoCloseTimeout)
```

Comment:
This is.. weirdly named. Should name it so it convey what it's *actually* doing. The original `CGUIDialogYesNo::ShowAndGetInput` much clearer in that regard.